src/main/io/vtx_smartaudio.c

```diff
@@ -761,16 +761,16 @@ void vtxSAProcess(timeUs_t currentTimeUs)
         // dprintf(("process: resending 0x%x\r\n", sa_outstanding));
         // XXX Todo: Resend termination and possible offline transition
         saResendCmd();
-	lastCommandSentMs = nowMs;
+    lastCommandSentMs = nowMs;
     } else if (!saQueueEmpty()) {
         // Command pending. Send it.
         // dprintf(("process: sending queue\r\n"));
         saSendQueue();
-	lastCommandSentMs = nowMs;
+    lastCommandSentMs = nowMs;
     } else if ((nowMs - lastCommandSentMs < SMARTAUDIO_POLLING_WINDOW) && (nowMs - sa_lastTransmissionMs >= SMARTAUDIO_POLLING_INTERVAL)) {
-	//dprintf(("process: sending status change polling\r\n"));
-	saGetSettings();
-	saSendQueue();
+    //dprintf(("process: sending status change polling\r\n"));
+    saGetSettings();
```

Comment:
It seems that not all indentation is good. It seems that the content has the same space as the { }.